account_payment_order/views/account_payment_line.xml

```diff
@@ -9,6 +9,7 @@
                     <group name="left">
                         <field
                             name="order_id"
+                            groups="base.group_multi_company"
                             invisible="not context.get('account_payment_line_main_view')"
                         />
                         <field name="name" />
```

Comment:
The proper way to fix this is adding an invisible `company_id` field that doesnt have the `groups="base.group_multi_company"` restriction. The comment you wrote in the PR about this error is to my knowledge some new behaviour in 16.0, but the tl;dr is that the `company_id` field in this view needs the multi-company group so it depends on that in order to use the `company_id` domain it adds automatically with `check_company`.

If you add `<field name="company_id" invisible="1" />` above it, it will work without this groups restriction, which is better since something like an order field shouldn't really be restricted to multi company

Reply:
Thanks - this is done